cmd/grafana-app-sdk/templates/manifest.cue.tmpl

```diff
@@ -1,5 +1,28 @@
 package kinds
 
+// config contains global code generation configuration options for the app. 
```

Comment:
I think it might be cleaner to put this into a new file called `config.cue` or `cli-config.cue` that we generate on project init.